etl-pipeline/model/source.py

```diff
@@ -5,6 +5,7 @@ class Source(Enum):
     CHICACO_ISAC = "isac"
     CLACSO = "clacso"
     DOAB = "doab"
+    GRIN = "grin"
```

Comment:
I think this is fine for now @priya-pai but we might want to consider maybe determining the source depending on the OCR package name or the like? maybe we should be using nypl as the source even if it's coming from GRIN. need to flush that out but we shall see